registry/handlers/api_test.go

```diff
@@ -2242,7 +2268,12 @@ func finishUpload(t *testing.T, ub *v2.URLBuilder, name reference.Named, uploadU
 	return resp.Header.Get("Location")
 }
 
-func doPushChunk(t *testing.T, uploadURLBase string, body io.Reader) (*http.Response, digest.Digest, error) {
+type chunkOption struct {
```

Comment:
Perhaps `chunkOptions` (plural) is a better name, to indicate this struct could have more than a single option at some point. However, also check my other comment.

Reply:
updated, thanks.